### ISSUE TITLE
Sync with SharePoint/OneDrive fails

### ISSUE THREAD

**Issue by jceb** [MasterSoft24/CloudCross#49]
I tried to connect CloudCross to a business account on O365 and didn't succeed.  Is it possible to synchronize with OneDrive on these `company.sharepoint.com` URLs?

**Comment by MasterSoft24** [comment]
No. Unfortunately, CloudCross works only with personal account yet.

**Comment by jceb** [comment]
Are there any plans on supporting sharepoint in the future?

**Comment by MasterSoft24** [comment]
Maybe in the future. I'm don't have any bussiness accounts in the inedrive
